src/cpp/rtps/builtin/data/WriterProxyData.cpp

```diff
@@ -242,79 +242,78 @@ uint32_t WriterProxyData::get_serialized_size(
         // PID_PERSISTENCE_GUID
         ret_val += 4 + PARAMETER_GUID_LENGTH;
     }
-    if (durability.send_always() || durability.hasChanged)
```

Comment:
Now that `hasChanged`, `send_always_` and `send_always()` are deprecated, would it be possible to clean up all references (basically assignments) to them?
Also, it may be a good idea to add a FASTDDS_TODO_BEFORE(4,..) remove them from `QosPolicy` class